worker/src/main/java/org/apache/celeborn/service/deploy/worker/storage/ChunkStreamManager.java

```diff
@@ -96,14 +96,14 @@ public TimeWindow getFetchTimeMetric(long streamId) {
     }
   }
 
-  public void chunkBeingSent(long streamId) {
+  public synchronized void chunkBeingSent(long streamId) {
     StreamState streamState = streams.get(streamId);
     if (streamState != null) {
       streamState.chunksBeingTransferred++;
     }
   }
```

Comment:
This will be effectively a global lock, right ?
If the concern is around inconsistent update to `chunksBeingTransferred` (which is valid), why not lock `streamState` instead ?

```suggestion
  public void chunkBeingSent(long streamId) {
    StreamState streamState = streams.get(streamId);
    if (streamState != null) {
      synchronized(streamState) {
        streamState.chunksBeingTransferred++;
      }
    }
  }
```

(same below)